.github/workflows/master.yml

```diff
@@ -0,0 +1,83 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2024 Intel Corporation
+
+name: Master workflow
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+env:
+  DOCKER_LABEL_VCS_URL: git remote get-url $(git remote)
+  VERSION: cat ./VERSION
+  DOCKER_LABEL_VCS_REF: git rev-parse HEAD
+  DOCKER_LABEL_BUILD_DATE: date -u "+%Y-%m-%dT%H:%M:%SZ"
+  DOCKER_LABEL_COMMIT_DATE: git show -s --format=%cd --date=iso-strict HEAD
+  DOCKER_BUILD_ARGS: --build-arg org_label_schema_version="$VERSION" --build-arg org_label_schema_vcs_url="$DOCKER_LABEL_VCS_URL" --build-arg org_label_schema_vcs_ref="$DOCKER_LABEL_VCS_REF" --build-arg org_label_schema_build_date="$DOCKER_LABEL_BUILD_DATE" --build-arg org_opencord_vcs_commit_date="$DOCKER_LABEL_COMMIT_DATE"
+  DOCKER_REPOSITORY: onosproject/
```

Comment:
Why do not you call `make images` or `make sdcore-adapter-docker` instead of adding all these environment variables here?